app/assets/javascripts/components/shared/modal.js

```diff
@@ -24,6 +24,11 @@
         showTitle: false,
         // Title of the modal – mandatory for accessibility
         title: 'Untitled modal',
+        // Allows the modal container have scroll
+        allowScroll: false,
```

Comment:
The scroll is blocked on Chrome and FF but not on Safari. This was my confusion.